Lena's attempt.Rmd

```diff
@@ -28,9 +28,13 @@ c.tree <- rpart(action ~ hint.y + complete, method="class", data=D1) #Notice the
 #Look at the error of this tree
 printcp(c.tree)
 
+plot(printcp(c.tree))
+
 #Plot the tree
 post(c.tree, file = "tree.ps", title = "Session Completion Action: 1 - Ask teacher, 2 - Start new session, 3 - Give up")
 
+#according to the graph, these two factors hint.y and complete are not much correlated with action
```

Comment:
I'm not sure what you mean by this? Also, the model doesn't demonstrate correlation.